src/Pos/Block/Logic/Slog.hs

```diff
@@ -159,14 +159,15 @@ slogApplyBlocks blunds = do
     -- Note: it's important to put blunds first
     mapM_ dbPutBlund blunds
     -- If the program is interrupted at this point (after putting on block),
-    -- we will rollback all wallet sets at the next launch.
-    onApplyBlocks blunds `catch` logWarn
+    -- we won't save any blocks.
+    callbackBlocks <- onApplyBlocks blunds
```

Comment:
@ksaric @pva701 I pushed a commit which makes comments and variable name more reasonable to me.